share/scripts/inverse/run_gromacs.sh

```diff
@@ -115,12 +117,32 @@ if [[ ${CSG_MDRUN_OPTS} ]]; then
   mdrun_opts+=" ${CSG_MDRUN_OPTS}"
 fi
 
+#support of REMD during gromacs simlation
+if [[ $(csg_get_property cg.inverse.gromacs.REMD) = "yes" ]]; then
+PREFIX="$(csg_get_property cg.inverse.gromacs.REMD.PREFIX)"
+read -a T <<<"$(csg_get_property cg.inverse.gromacs.REMD.T_for_replicas)"
+j=0
+for i in "${T[@]}"
+do
+echo $i
+#TEMP has to be written as a place holer for the temperature in the *.mdp file
+sed "s/TEMP/$i/" grompp.mdp > grompp_$j.mdp
+critical ${grompp[@]} -n "${index}" -f "grompp_$j.mdp" -p "$topol_in" -o "$PREFIX"_"$j.tpr" -c "${conf}" ${grompp_opts} 2>&1 | gromacs_log "${grompp[@]} -n "${index}" -f "${mdp}" -p "$topol_in" -o "$tpr" -c "${conf}" ${grompp_opts}"
```

Comment:
Hi both, I am trying to follow what is done here: 

1) replica-exchange 'REMD' option is searched in options.xml by the tag 'cg.inverse.gromacs.REMD' and, if it's found with value "yes", then an REMD branch in run_gromacs.sh is followed (this is clear, just for the sake of completeness)

2) some PREFIX is read in from the tag 'cg.inverse.gromacs.REMD.PREFIX' - what is this prefix? - It looks like "$PREFIX" plus the replica index is the .tpr file name; Why calling the tag 'PREFIX' then? - would it be clearer (self-explanatory for the user) to call it what it is: 'REMD.tpr_name'? It does not seem that PREFIX is used as anything else(?). As it is, at first look I was anticipating it to be a prefix for some standard file name(s).

3) A set of Temperatures is read in from options.xml tag 'cg.inverse.gromacs.REMD.T_for_replicas' - to my liking 'T_for_replicas' is too long for a tag, e.g. I would prefer 'replicas_T', but that's a matter of taste of course

4) Using template grompp.mdp file: each T value is added into it by replacing the place-holder "TEMP" (if it's found in grompp.mdp) and saving under the name grompp_$j.mdp (suffix being the replica index). 

What I dislike: grompp.mdp needs to have a place-holder TEMP to be replaced with an actual option, i.e. this grompp cannot be used by itself, without modification. Even though it is just one line, the user will have to remember and take care of that line. - as an alternative, I would replace the existing(!) option 'ref-t ='*  in grompp.mdp:

sed '/ref-t/s/.*/$i/' grompp.mdp > grompp_$j.mdp

Another thin is that "grompp" is hard-coded as the file name, whereas the user might what to have some generic name-pattern for all the files: my_name.mdp, my_name.top, my_name.gro etc.

Sorry for this being so long / Andrey